src/lang/modifyAst/addFillet.ts

```diff
@@ -147,64 +169,89 @@ export function modifyAstWithFilletAndTag(
     if (err(locatedExtrudeDeclarator)) return locatedExtrudeDeclarator
     const { extrudeDeclarator } = locatedExtrudeDeclarator
 
-    // Modify the extrude expression to include this fillet expression
-    // CallExpression - no fillet
-    // PipeExpression - fillet exists or extrude in sketch pipe
+    // Modify the extrude expression to include this edge treatment expression
+    // CallExpression - no edge treatment
+    // PipeExpression - edge treatment exists or body in sketch pipe
 
-    let pathToFilletNode: PathToNode = []
+    let pathToEdgeTreatmentNode: PathToNode = []
```

Comment:
The assignment of dumby data (`[]`) is not needed as each branch of the if else statement either assigns `pathToEdgeTreatmentNode` or returns early.
```suggestion
    let pathToEdgeTreatmentNode: PathToNode
```
tsc should be fine with the above.